config/config.go

```diff
@@ -613,9 +613,32 @@ func ValidateChanges(oldCfg, newCfg *Config) error {
 	if err := ValidateNamespaceSyncChanges(oldCfg, newCfg); err != nil { //nolint:revive
 		return err
 	}
+
+	return nil
+}
+
+// ValidateProjectConfigChanges checks for disallowed config changes in project
+func ValidateProjectConfigChanges(oldCfg, newCfg *Config) error {
+	if oldCfg.PlatformProjectSet() && newCfg.PlatformProjectSet() {
```

Comment:
Don't think we can check if like that, that just compares the configured project in a single place, the actual comparison needs to take place when the virtual cluster instance is created